docs/source/apps/supermarq/qcvv/qcvv_irb_css.ipynb

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
<p>Style nit: add <code>supermarq</code> import here (and maybe put all imports in a separate cell)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/994/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>